test/distributed/elastic/agent/server/test/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
```

Comment:
These files are already under the top level test/ directory, do we need another test/ suffix at the end?